### PR TITLE
fix(libinput): read display dimensions from driver

### PR DESCRIPTION
**Pull request by Johennes**
This changes the libinput driver to read unrotated display dimensions directly from the driver. This is necessary because libinput itself doesn't respect any rotation inside of LVGL. As a result, coordinates retrieved from libinput need to be compared to the unrotated dimensions.

Fixes: #188

### CODE REVIEW

File: indev/libinput.c [hunk 1]
```diff
@@ -397,10 +402,8 @@ static void read_pointer(libinput_drv_state_t *state, struct libinput_event *eve
       pointer_event = libinput_event_get_pointer_event(event);
       state->most_recent_touch_point.x += libinput_event_pointer_get_dx(pointer_event);
       state->most_recent_touch_point.y += libinput_event_pointer_get_dy(pointer_event);
-      state->most_recent_touch_point.x = state->most_recent_touch_point.x < 0 ? 0 : state->most_recent_touch_point.x;
-      state->most_recent_touch_point.x = state->most_recent_touch_point.x > LV_HOR_RES - 1 ? LV_HOR_RES - 1 : state->most_recent_touch_point.x;
-      state->most_recent_touch_point.y = state->most_recent_touch_point.y < 0 ? 0 : state->most_recent_touch_point.y;
-      state->most_recent_touch_point.y = state->most_recent_touch_point.y > LV_VER_RES - 1 ? LV_VER_RES - 1 : state->most_recent_touch_point.y;
+      state->most_recent_touch_point.x = LV_MIN(LV_MAX(state->most_recent_touch_point.x, 0), drv->hor_res - 1);
```
Comment (kisvegabor) [inline indev/libinput.c:405]:
```suggestion
      state->most_recent_touch_point.x = LV_CLAMP(0, state->most_recent_touch_point.x, drv->hor_res - 1);
```

**Review by kisvegabor — COMMENTED** [review]
Looks good, only a minor suggestion.

File: indev/libinput.c (hunk 1 above)
Comment (Johennes) [reply indev/libinput.c]: Ohh, I missed the `LV_CLAMP` macro. :facepalm: 

Applied the fix.

**Review by embeddedt — APPROVED** [review]

**Comment by embeddedt** [comment]
Merged, thanks!

**Pull request merged**